symbolic-debuginfo/src/breakpad.rs

```diff
@@ -36,28 +24,34 @@ const UNKNOWN_NAME: &str = "<unknown>";
 
 /// The error type for [`BreakpadError`].
 #[non_exhaustive]
-#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
 pub enum BreakpadErrorKind {
     /// The symbol header (`MODULE` record) is missing.
     InvalidMagic,
 
     /// A part of the file is not encoded in valid UTF-8.
     BadEncoding,
 
-    /// A record violates the Breakpad symbol syntax.
-    BadSyntax,
```

Comment:
same comment as on the other PR: this type is public, so you must not remove or change any variants, even if you stop using them.

Reply:
Goddammit, I knew I forgot something ^^ Since it's not used, should I add some sort of deprecation doc comment?

Reply:
sure, there is a deprecation attribute for that